plugins/modules/aci_bd.py

```diff
@@ -147,6 +149,58 @@
     description:
     - The L3 Out that contains the associated Route Profile.
     type: str
+  host_based_routing:
+    description:
+    - Enables advertising host routes (/32 prefixes) out of the L3OUT(s) that are associated to this BD.
+    - The APIC defaults to C(false) when unset during creation.
+    type: bool
+    aliases: [ advertise_host_routes ]
+  enable_rogue_except_mac:
+    description:
+    - Rogue exception MAC wildcard support for Bridge Domains.
+    - Only available in APIC version 6.0 or later.
+    - The APIC defaults to C(false) when unset during creation.
+    type: bool
+  allow_intersite_bum_traffic:
+    description:
+    - Control whether BUM traffic is allowed between sites.
+    - The APIC defaults to C(false) when unset during creation.
+    type: bool
+    aliases: [allow_bum_traffic]
+  allow_intersite_l2_stretch:
+    description:
+    - Allow L2 Stretch between sites.
+    - The APIC defaults to C(false) when unset during creation.
+    type: bool
+    aliases: [allow_l2_stretch]
+  allow_ipv6_mcast:
+    description:
+    - Flag to indicate if ipv6 multicast is enabled.
+    - The APIC defaults to C(false) when unset during creation.
+    type: bool
+  ll_addr:
+    description:
+    - The override of the system generated IPv6 link-local address.
+    type: str
+    aliases: [ll_addr_ipv6]
+  mcast_arp_drop:
+    description:
+    - Enable BD rogue multicast ARP packet drop.
+    - Only available in APIC version 6.0 or later.
+    - The APIC defaults to C(true) when unset during creation.
+    type: bool
+  ipv6_unk_mcast_act:
+    description:
+    - Unknown IPv6 Multicast Destination Action.
+    - The APIC defaults to C(flood) when unset during creation.
+    type: str
+    choices: [ flood, opt-flood ]
+    aliases: [v6unk_mcast_act]
```

Comment:
should we add some aliases for this one?
```suggestion
  ipv6_unknown_multicast_action:
    description:
    - Unknown IPv6 Multicast Destination Action.
    - The APIC defaults to C(flood) when unset during creation.
    type: str
    choices: [ flood, opt-flood ]
    aliases: [ ipv6_unk_mcast_act ]
```